### ISSUE TITLE
[Python] requirements.txt empty on new projects

### ISSUE THREAD

**Issue by jeffhollan** [Microsoft/vscode-azurefunctions#1178]
Realize this may be on another repo or the template feed - tried to poke around and couldn't figure out where it's pulled from, but today the project template for python results in a `requirements.txt` file that is empty.  This seems a bit strange to me as the project has `import azure.functions as func` which my VS Code instance can't give me nice intellisense for unless I have `azure-functions` as one of the pip packages to install.

Seems we should have the default project template include `azure-functions` as a single package to import by default.

**Comment by EricJizbaMSFT** [comment]
Yeah we use the func cli to create python projects. Moved here: https://github.com/Azure/azure-functions-core-tools/issues/1194
